modes/company/evil-collection-company.el

```diff
@@ -112,6 +126,9 @@ C-x C-l."
     (kbd "M-k") 'company-select-previous
     (kbd "<escape>") 'company-search-abort)
 
+  ;; https://github.com/emacs-evil/evil-collection/issues/664
+  (add-hook 'evil-local-mode-hook #'evil-collection-company-protect-keymap)
```

Comment:
I'll probably move the toggle around this add-hook instead at some point if I remember. Thanks!